src/main/pages/che-7/administration-guide/assembly_configuring-authorization.adoc

```diff
@@ -0,0 +1,123 @@
+---
+title: "Configuring authorization"
+keywords: organizations, user management, permissions
+tags: [ldap, keycloak]
+sidebar: che_7_docs
+permalink: che-7/configuring-authorization/
+folder: che-7/administration-guide
+summary: 
+---
+
+:parent-context-of-configuring-authorization: {context}
+
+:context: configuring-authorization
+
+[id="authorization-and-user-management"]
+== Authorization and user management
+
+Eclipse Che uses http://www.Keycloak.org[Keycloak] to create, import, manage, delete, and authenticate users.  Keycloak uses built-in authentication mechanisms and user storage.  It can use third-party identity management systems to create and authenticate users. Eclipse Che requires a Keycloak token when you request access to Che resources.
+
+Local users and imported federation users must have an email address in their profile.
+
+The default Keycloak credentials are `admin:admin`. You can use the `admin:admin` credentials when logging into Che for the first time. It has system privileges. 
+
+To find your Keycloak URL:
+
+If Che is deployed on OpenShift:
+
+* Go to the OpenShift web console and navigate to the *Keycloak* namespace. 
+
+ifeval::["{project-context}" == "che"]
+If Che is running on Docker:
+
+* Go to `$CHE_HOST:5050/auth`.
+endif::[]
+
+
+[id="configuring-che-to-work-with-keycloak"]
+== Configuring Che to work with Keycloak
+
+The deployment script ensures that Keycloak is properly configured when Che is deployed on OpenShift or installed on Docker. When the `che-public` client is created, the following fields are populated:
+
+* *Valid Redirect URIs*: Use this URL to access Che. 
+* *Web Origins*
+
+The following are common errors when configuring Che to work with Keycloak: 
+
+*Invalid `redirectURI` error*:  occurs when you access Che at `myhost`, which is an alias, and your original `CHE_HOST` is `1.1.1.1`.  If this error occurs, go to the Keycloak administration console and ensure that the valid redirect URIs are configured.
+
+*CORS error*: occurs when you have an invalid web origin
+
+
+[id="configuring-keycloak-tokens"]
+== Configuring Keycloak tokens
+
+A user token expires after 30 minutes by default. 
+
+You can change the following Keycloak token settings:
```

Comment:
@skabashnyuk and @mshaposhnik : If I'm not mistaken there is also a Che property that can configure that. Is that correct? In this case, I think it should be inline with the content here.

Reply:
There is no ```Che property``` that can configure that. 
It is set in configuration https://github.com/eclipse/che/blob/master/dockerfiles/keycloak/che-realm.json.erb#L8-L9